ratpack-rx/src/main/java/ratpack/rx/RxRatpack.java

```diff
@@ -346,6 +330,7 @@ public static void initialize() {
    * @see #promise(Observable)
    * @throws UnmanagedThreadException if called outside of an execution
    */
+  @Deprecated
```

Comment:
should we deprecate these methods?  the `rx.Observable.extend` method is deprecated and marked for removal in RxJava 1.2, but that appears to be the primary use case for these.